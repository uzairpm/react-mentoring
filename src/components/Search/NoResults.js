import React from 'react';

const NoResults = (props) => {
    return (
        <h1 className="display-3 text-center mt-5">No films found</h1>
    );
};
export default NoResults;
