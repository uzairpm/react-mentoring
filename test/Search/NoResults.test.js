import React from 'react';

import NoResults from '../../src/components/Search/NoResults';

describe('Check Component creation', () => {
    test('NoResults component', () => {
        const element = shallow(<NoResults />);
        // console.log(element.debug());
        expect(element).toMatchSnapshot();
    });
});
