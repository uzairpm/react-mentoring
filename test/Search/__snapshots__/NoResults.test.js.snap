// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check Component creation NoResults component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NoResults />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "No films found",
      "className": "display-3 text-center mt-5",
    },
    "ref": null,
    "rendered": "No films found",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "No films found",
        "className": "display-3 text-center mt-5",
      },
      "ref": null,
      "rendered": "No films found",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
